controllers/azureidentity_controller.go

```diff
@@ -83,9 +80,9 @@ func (r *AzureIdentityReconciler) SetupWithManager(mgr ctrl.Manager, options con
 // +kubebuilder:rbac:groups="",resources=secrets;,verbs=get;list;watch
 
 // Reconcile reconciles the Azure identity.
-func (r *AzureIdentityReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr error) {
-	ctx, cancel := context.WithTimeout(context.Background(), reconciler.DefaultedLoopTimeout(r.ReconcileTimeout))
-	defer cancel()
+func (r *AzureIdentityReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, reterr error) {
+	//ctx, cancel := context.WithTimeout(context.Background(), reconciler.DefaultedLoopTimeout(r.ReconcileTimeout))
+	//defer cancel()
```

Comment:
not sure what if this is the right thing to do here with context since it now has to be in the signature of the func, in capi they set it in main.go and in this case here was getting an error saying ctx is being updated on its first use

@devigned @CecileRobertMichon

Reply:
Same as previous comment on context.

Reply:
fixed